api/HTMLElement.json

```
@@ -2862,6 +2862,41 @@
             "deprecated": false
           }
         },
+        "force": {
```

I copied the pattern of `source`.
For both this and `source` shouldn't the actual pattern be as below, based on the schema? Or did that change?

```suggestion
        "option_force": {
```

---

Given that `force` can be both a parameter and an option, and the option came later, we might want to record it as both `force_parameter` and `force_option`, to avoid confusion.